docfx/articles/saveload.md

```diff
@@ -46,24 +42,32 @@ For efficient memory management, the model should be created on the CPU before l
 
 ><br/>It is __critical__ that all submodules and buffers in a custom module or composed by a Sequential object have exactly the same name in the original and target models, since that is how persisted tensors are associated with the model into which they are loaded.<br/><br/>The CustomModule 'RegisterComponents' will automatically find all fields that are either modules or tensors, register the former as modules, and the latter as buffers. It registers all of these using the name of the field, just like the PyTorch Module base class does.<br/><br/>
 
-### Saving a TorchSharp format model in Python
+### How to use the PyTorch format
```

Comment:
It looks like you're replacing the TorchSharp-specific instructions pickling-based instructions. I would advice you to add rather than replace.

Reply:
We also want to make sure that we know it's something that has at least the same level of support that TorchSharp does.

Reply:
Sure, of course. 
This feature is meant to bypass the need for the `importsd.py` and `exportsd.py`, so I had replaced the instructions that used those scripts. 
I'll rewrite the changes so that I'm adding and leaving in instructions for using the script for conversion and saving using the (new) built-in `module.save_py` and `module.load_py` methods.

Reply:
> We also want to make sure that we know it's something that has at least the same level of support that TorchSharp does.

This was meant as a response to my question about the new dependency, not the article..

Reply:
> > We also want to make sure that we know it's something that has at least the same level of support that TorchSharp does.
> 
> This was meant as a response to my question about the new dependency, not the article..

Ah! Now I understand.
It seems to be up to date - supporting the latest pickle protocol, but I'll leave that judgment up to you.